src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -5598,6 +5602,59 @@ CmdSketcherConstrainPerpendicular::CmdSketcherConstrainPerpendicular()
     ;
 }
 
+bool CmdSketcherConstrainPerpendicular::removeRedundantPointOnObject(SketchObject* Obj,
```

Comment:
I would very much like to remove the duplication of this method three times.

Reply:
Should be done.